.eslint-resolver.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+const assign = require ( 'lodash/assign' );
+const debugFactory = require( 'debug' );
+const nodeResolver = require( 'eslint-import-resolver-node' );
+const path = require( 'path' );
+const webpackResolver = require( 'eslint-import-resolver-webpack' );
+
+/**
+ * Internal dependencies
+ */
+const webpackResolveConfig = require( './webpack.config.resolve' );
+
+/**
+ * Module variables
+ */
+const log = debugFactory('eslint-plugin-import:resolver:wp-calypso-resolver');
+
+const nodeResolverConfig = {
+	extensions: [
+		'.js',
+		'.jsx',
+	],
+	moduleDirectory: [
+		'node_modules',
+		'server'
+	],
+	paths: [
+		path.join( __dirname, 'test' ),
+		path.join( __dirname, 'server' ),
+		path.join( __dirname, 'client' ),
+	]
+};
+
+const webpackResolverConfig = {
+	config: {
+		resolve: webpackResolveConfig,
+	},
+};
+
+exports.interfaceVersion = 2
+
+exports.resolve = function (source, file, config) {
```

Comment:
I realized that ESLint was ignoring the file due to the `.eslint` prefix. I went ahead and changed `.eslintignore` to force parsing `.eslint-resolver` so this shouldn't happen again.